openBCIBoard.js

```diff
@@ -172,10 +207,9 @@ function OpenBCIFactory() {
     OpenBCIBoard.prototype.disconnect = function() {
         var timeout = 0;
         if (this.streaming) {
+            this.streamStop();
             if(this.options.verbose) console.log('stop streaming');
-            this.streaming = false;
-            this.write(k.OBCIStreamStop);
-            timeout = 10;
+            timeout = 60;
```

Comment:
you could have a helper function that evaluates the state of the board
(`if streaming time = x else y`) i'm always apprehensive about hard-coded values lol